doc/api/url.md

```diff
@@ -317,6 +317,15 @@ console.log(myURL.port);
 myURL.port = 1e10;
 console.log(myURL.port);
 // Prints 1234
+
+// Out-of-range numbers, which are converted to exponential 
+// notation via .toString(), will behave as strings with leading zeroes.
+// This means that the port will be assigned the integer part of the coefficient,
+// assuming the number is normalized (for example, 0.9e10 => 9e9).
+// See https://www.ecma-international.org/ecma-262/6.0/#sec-tostring-applied-to-the-number-type for more information
```

Comment:
I'd omit the spec link. I also think "will behave as strings with leading zeroes" might be more confusing than clarifying.

Reply:
Fixed, and made the entire thing arguably clearer